pkg/loadbalancer/experimental/reflector.go

```diff
@@ -153,8 +154,6 @@ func runResourceReflector(ctx context.Context, p reflectorParams, initComplete f
 			}
 			txn := p.Writer.WriteTxn()
 			for _, ev := range buf {
-				ev.Done(nil)
```

Comment:
This change looks unrelated to the PR. What's the rationale behind it?

Reply:
> I can move the change into its own commit if you'd like.

That would be great, thanks!

Reply:
Done